webview.h

```diff
@@ -2361,6 +2431,13 @@ class win32_edge_engine {
     return scale_size(size.cx, size.cy, from_dpi, to_dpi);
   }
 
+  void on_system_setting_change(const wchar_t *area) {
+    // Detect light/dark mode change in system.
+    if (lstrcmpW(area, L"ImmersiveColorSet") == 0) {
+      apply_window_theme(m_window);
```

Comment:
When the theme changes system-wide then we receive `WM_SETTINGCHANGE` multiple times rapidly. This is unfortunate because we'll attempt to apply the new theme to the window with the same frequency.

Reply:
Changing the setting on the file happens fairly rarely. And the implementation works well 🙂